Fabric.Identity.API/scripts/Install-IdentityProviderSearchService.ps1

```diff
@@ -0,0 +1,85 @@
+﻿param(
+    [PSCredential] $credential, 
+    [Hashtable] $configStore = @{Type = "File"; Format = "XML"; Path = "$PSScriptRoot\install.config"},
+    [Hashtable] $certificates,
```

Comment:
Instead of passing this in, can we get this from the common config?

Reply:
I think it is the $certificates that you are asking about, if not let me know. The last method Set-IdentityProviderSearchServiceWebConfigSettings requires the SigningCertificate and the EncryptionCertificate.Thumbprint. We only have the primary and encryption thumbprints in common.config. The signing certificate looks like a version of the PrimarySigningCertificateThumbprint with a regex replace. I guess I could call Get-Certificates again, but I thought I tried that and had an issue.